lib/src/exception/exception.test.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {URL} from 'url';
+
+import {SassException} from './exception';
+import {SourceSpan} from './span';
+
+describe('SassException', () => {
+  it('has the correct type', () => {
+    const error = new SassException({message: ''});
+
+    try {
+      throw error;
+    } catch (error) {
+      expect(error instanceof SassException).toBe(true);
+    }
```

Comment:
I'm pretty sure this much is guaranteed by the JS language, so you probably don't need to test it explicitly.